x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/collector/Collector.java

```diff
@@ -84,7 +84,8 @@ protected boolean shouldCollect(final boolean isElectedMaster) {
                 return doCollect(convertNode(timestamp, clusterService.localNode()), interval, clusterState);
             }
         } catch (ElasticsearchTimeoutException e) {
-            logger.error((Supplier<?>) () -> new ParameterizedMessage("collector [{}] timed out when collecting data", name()));
+            logger.error((Supplier<?>) () ->
+                    new ParameterizedMessage("collector [{}] timed out when collecting data: {}", name(), e.getMessage()));
```

Comment:
(super minor nit)

Why do we use a `ParameterizedMessage` in the any case here, since the exception is not actually passed? Could this be instead

```suggestion
            logger.error("collector [{}] timed out when collecting data: {}", name(), e.getMessage());
```

?

Reply:
It's a valid question, I don't know, I was just making minimal adjustments to the existing code. Fixed in f68ef2e.